src/ui/screens.ts

```diff
@@ -0,0 +1,433 @@
+    const folders = [];
+
+    do {
+        joplinFolders = await joplin.data.get(['folders']);
+
+        for (let i = 0; i < joplinFolders.items.length; i++) {
+            const folder = joplinFolders.items[i];
+            let path = folder.title;
+            let node = folder;
+            const id = folder.id;
+
+            // The parent folder path
+            while (node.parent_id !== '') {
+                node = await joplin.data.get(['folders', node.parent_id]);
+                path = `${node.title}/${path}`;
+            }
+            folders.push({path: path, id: id});
+        }
+    } while (joplinFolders.has_more);
+
+    const options = [];
+
+    folders.forEach((folder) => {
+        const option = lastState['folderId'] === folder.id ? `<option selected value="${folder.id}">${folder.path}</option>` : `<option value="${folder.id}">${folder.path}</option>`;
+        options.push(option);
+    });
+
+
+    const mailBoxesOptions = [];
+    lastState['mailBoxes'].forEach((element) => {
+        const option = lastState['mailBox'] === element.path ? `<option selected value="${element.value}">${element.path}</option>`: `<option value="${element.value}">${element.path}</option>`;
+        mailBoxesOptions.push(option);
+    });
+
+
+    return `
+<div class="container">
+
+<div class="row" style="font-size: large;">
+
+  <div class="col-md-9 col-lg-7 col-xl-5 mx-auto">
+
+  <div class="card border-0 shadow rounded-3 my-3 align-middle d-flex justify-content-center" style="opacity: 0.97; top: 100px;" >
+
+      <div class="card-body p-4 p-sm-5">
+
+        <h1 class="card-title text-center mb-3 fw-light fs-1">Main Screen</h1>
+
+        <div class="container" style="text-align: center;">
+
+          <div class="input-group mb-3">
+            <span class="input-group-text" id="basic-addon1">From</span>
+            <input type="email" class="form-control" placeholder="email" aria-label="Email"
+              aria-describedby="basic-addon1" id='from' value = "${fromValue}" ${readOnly} required>
+          </div>
+          <div class="container" style="text-align: center">
+            <div class="form-check form-switch">
+              <input class="form-check-input" type="checkbox" role="switch" id="toggle-from"
+                onchange="toggle()" ${checked}>
+              <label class="form-check-label" for="toggle-from">Fetching & Monitoring</label>
+            </div>
+          </div>
+        </div>
+
+        <br>
+
+        <div class="container" style="text-align: center;">
+          <div class="input-group mb-3">  
+            <span class="input-group-text" id="basic-addon1">MailBoxes</span>
+            <select class="form-select" aria-label="Default select example" id = 'mailbox' ${disabledMailBoxes}>
+              ${mailBoxesOptions}
+            </select>
+          </div>
+
+        <div class="input-group mb-3">
+          <span class="input-group-text" id="basic-addon1"> NoteBooks :</span>
```

Comment:
minor thing but formatting here is inconsistent - MailBoxes does not have a column, NoteBooks does (and with a space in between!)
Notebooks should be one word, I think, not camel case.

Reply:
UPD removing the colon makes things look a bit nicer (for me anyway)

Reply:
I fixed it and this is the final result. If you want another modification, please tell me.
![f1](https://user-images.githubusercontent.com/58605547/188417343-f2f91577-69e2-4815-9902-d6c5a9ad7238.png)

Reply:
Looks better, thanks.
Ideally I it feels to me the labels should have equal widths but it probably isn't the best use of time now. Up to you.

Reply:
Done, I equalised the width of the two labels.

![lastMS](https://user-images.githubusercontent.com/58605547/188584169-5cc1c768-f073-4cb1-8775-7806977b083b.png)